ingesters/auditlog/auditlogingester_test.go

```diff
@@ -0,0 +1,69 @@
+package auditlog_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+
+	"github.com/metal-toolbox/audito-maldito/ingesters/auditlog"
+	"github.com/metal-toolbox/audito-maldito/ingesters/namedpipe"
+	"github.com/metal-toolbox/audito-maldito/internal/health"
+)
+
+func TestIngest(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+
+	pipePath := fmt.Sprintf("%s/audit-pipe", tmpDir)
+	err := syscall.Mkfifo(pipePath, 0o664)
+	if err != nil {
+		assert.Error(t, err, "failed to initialize tests: Could not create %s/%s named pipe", tmpDir, pipePath)
+	}
```

Comment:
```go
	if err != nil {
		assert.Error(t, err, "failed to initialize tests: Could not create %s/%s named pipe", tmpDir, pipePath)
	}
```

Should be replaced with

```go
assert.NoError(t, err, "failed to initialize tests: Could not create %s/%s named pipe", tmpDir, pipePath)
```

The intent is that it's declarative and you shouldn't need if/else's for this.